packages/graphql-types-generator/test/__snapshots__/loading.ts.snap

```diff
@@ -1,389 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This snapshot is for a `Document` type and is very difficult to read. It is platform dependent (Windows/Linux) because of the `end, start` keys in the document.